internal/pkg/cli/deploy/deploy_test.go

```diff
@@ -128,6 +160,139 @@ func TestWorkloadDeployer_UploadArtifacts(t *testing.T) {
 			},
 			wantImageDigest: aws.String("mockDigest"),
 		},
+		"should retrieve Load Balanced Web Service custom resource URLs": {
+			inUploadCustomResourcesFlag: true,
+			mock: func(m *deployMocks) {
+				// Ignore addon uploads.
+				m.mockTemplater.EXPECT().Template().Return("", &addon.ErrAddonsNotFound{})
+
+				// Ensure all custom resources were uploaded.
+				crs, err := customresource.LBWS(fakeTemplateFS())
+				require.NoError(t, err)
```

Comment:
will this fail the parent test, not the subtest? (idk what the output looks like, but if it's clear which test failed, then idc)

Reply:
yeah it's pretty awkward that it causes the parent test to fail :rip: we should take `*testing.T` as an input, let me see if I can fix that real quick